source/AI.cpp

```diff
@@ -1945,7 +1945,10 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition,
 	bool isFacing = (dp.Unit().Dot(angle.Unit()) > .95);
 	if(!isClose || (!isFacing && !shouldReverse))
 		command.SetTurn(TurnToward(ship, dp, 0.9999));
-	if(isFacing)
+	// Work with a slightly lower maximum velocity to avoid border cases.
```

Comment:
```suggestion
	// Drag is not applied when not thrusting, so stop thrusting when close to max speed
	// to save energy. Work with a slightly lower maximum velocity to avoid border cases.
```